cgmanifests/generate_cgmanifest.py

```diff
@@ -73,7 +73,7 @@ def add_github_dep(name, parsed_url):
             return
         # Make a REST call to convert to tag to a git commit
         url = f"https://api.github.com/repos/{org_name}/{repo_name}/git/refs/tags/{tag}"
-        print("requesting %s ..." % url)
+        print("requesting {url} ...")
```

Comment:
This change is not right. Now it prints the string as it is.
```
requesting {url} ...
```
@justinchuby

Reply:
I forgot an f""

Reply:
I will fix forward

Reply:
https://github.com/microsoft/onnxruntime/pull/21489